flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/PartitionLoader.java

```diff
@@ -52,17 +52,27 @@ public class PartitionLoader implements Closeable {
     private final TableMetaStoreFactory.TableMetaStore metaStore;
     // whether it's to load files to local file system
     private final boolean isToLocal;
+    private final boolean writeSuccessFileEnabled;
```

Comment:
May not need to keep this field in this class. When commitPolicy is not null, apply this policy.